tjreports/site/models/reports.php

```diff
@@ -637,6 +650,7 @@ protected function populateState($ordering = '', $direction = 'ASC')
 	 */
 	protected function getListQuery()
 	{
+		$input 			= Factory::getApplication()->input;
```

Comment:
@pranotiTechjoomla Don't use the inputs here. Use model state

Reply:
Input is used to get a layout from URL of ajax call